lib/puppet/type/group.rb

```diff
@@ -80,7 +80,9 @@ def sync
 
     newproperty(:members, :array_matching => :all, :required_features => :manages_members) do
       desc "The members of the group. For directory services where group
-      membership is stored in the group objects, not the users."
+      membership is stored in the group objects, not the users. Combine
+      with auth_membership to determine whether the specified members
```

Comment:
Would "Use auth_membership to..." work? Combine suggests something more complex than just setting a parameter.